LICENCE.md

```diff
@@ -0,0 +1,61 @@
+# Document Licence
+
+## Licence
+
+By using and/or copying this document, or the IDRC document from which this
+statement is linked, you (the Licensee) agree that you have read, understood,
+and will comply with the following terms and conditions:
+
+Permission to copy, and distribute the contents of this document, or the IDRC
+document from which this statement is linked, in any medium for any purpose and
+without fee or royalty is hereby granted, provided that you include the
+following on ALL copies of the document, or portions thereof, that you use:
+
+- A link or URL to the original IDRC document.
+- The pre-existing copyright notice of the original author, or if it doesn't
+  exist, a notice (hypertext is preferred, but a textual representation is
+  permitted) of the form: "Copyright © [$date-of-document] World Wide Web
+  Consortium. https://github.com/inclusive-design/trust-meter/LICENCE.md"
+- If it exists, the STATUS of the IDRC document.
+
+When space permits, inclusion of the full text of this NOTICE should be
+provided. We request that authorship attribution be provided in any software,
+documents, or other items or products that you create pursuant to the
+implementation of the contents of this document, or any portion thereof.
+
+No right to create modifications or derivatives of IDRC documents is granted
+pursuant to this license, except as follows: To facilitate implementation of the
+technical specifications set forth in this document, anyone may prepare and
+distribute derivative works and portions of this document in software, in
+supporting materials accompanying software, and in documentation of software,
+PROVIDED that all such works include the notice below. HOWEVER, the publication
+of derivative works of this document for use as a technical specification is
+expressly prohibited.
+
+In addition, "Code Components" —Web IDL in sections clearly marked as Web IDL;
+and IDRC-defined markup (HTML, CSS, etc.) and computer programming language code
+clearly marked as code examples— are licensed under the ???? Licence.
```

Comment:
Seems we need to update this section with specific information.

Reply:
RIght, and it leads to other questions:
- will the technical specification include "Code Components"?  I wouldn't want to preclude it.
- what software licence is used for that aspect?